builtin/fsmonitor--daemon.c

```diff
@@ -1438,6 +1439,13 @@ int cmd_fsmonitor__daemon(int argc, const char **argv, const char *prefix)
 #else
 int cmd_fsmonitor__daemon(int argc, const char **argv, const char *prefix)
 {
+	struct option options[] = {
+		OPT_END(),
+	};
+
+	if (argc == 2 && !strcmp(argv[1], "-h"))
+		usage_with_options(builtin_fsmonitor__daemon_usage, options);
+
```

Comment:
This should probably be squashed into the commit that adds `builtin/fsmonitor--daemon.c`, and for readability I would switch it to an `#ifndef HAVE_FSMONITOR_DAEMON_BACKEND` and have the shorter conditional arm come first.

Reply:
Stolee added that commit into the microsoft/git branch when he was preparing the experimental release.  i dropped it on top of my fsmonitor3 branch for completeness.

My fsmonitor4 branch already has this squashed down into the commit that added the surrounding code.

Reply:
I should close this fsmonitor3 branch on GGG.  I was going to create a new one for fsmonitor4 when i get ready to send part 3 to the mailing list.

Reply:
I've already done those merges in the fsmonitor4 branch.  (sorry for the confusion.)

Reply:
Of course you did, and I even saw that! My bad, sorry.

Reply:
Hey @jeffhostetler! Now that `jh/simple-ipc` made it into `next`, you could rebase this PR onto `next` (if you hit the `Edit` button next to the PR title, you can change the target branch). What do you think?

Reply:
um, I should rebase fsmonitor4 onto next and get rid of those merge commits that i added.  i've been holding fsmonitor3 constant to reflect the experimental release that we made.

I do have a few commits to push to simple-ipc for some last minute change suggestions from Friday.  i should be able to do those in parallel.  i'll look at doing that tomorrow.  (my day got away from me today.)